dts/bindings/sensor/bosch,bma280.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Bosch BMA280 triaxial acceleration sensor.  See more info at:
+  https://www.bosch-sensortec.com/products/motion-sensors/accelerometers/bma280.html
+
+compatible: "bosch,bma280"
+
+include: i2c-device.yaml
+
+properties:
+  int1-gpios:
+    type: phandle-array
+    required: false
+    description: |
+      Identifies pin for the INT1 signal on the sensor.  The sensor
+      INT2 signal is not supported by the driver.
+
+  is-bmc150:
+    type: boolean
+    required: false
+    description: |
+      Specifies that the driver should adapt to the accelerometer
+      capability provided by the presented by the BMC150 6-axis
```

Comment:
Description typo. Should read "... accelerometer capability provided by the BMC150 6-axis ..."

Reply:
Revised, thanks.